mix.exs

```diff
@@ -24,14 +25,14 @@ defmodule Etude.Mixfile do
   end
 
   defp deps do
-    [{:rebind, "~> 0.1.0"},
+    [{:rebind, "~> 0.1.3"},
      {:lineo, "~> 0.1.0"},
      # hex doesn't publish github deps. i'd prefer uwiger publish parse_trans.
      # anyone that uses etude will need to add this manually for now
```

Comment:
Oops, didn't think to verify that travis worked.  That's fixed, as is the comment cleanup.  I rebased and recommitted.